embedded-graphics/src/image/image16bpp.rs

```diff
@@ -21,87 +18,27 @@ use core::marker::PhantomData;
 /// ```
 /// This will remove the BMP header leaving the raw pixel data
 /// E.g 64x64 image will have `64 * 64 * 2` bytes of raw data.
-#[derive(Debug)]
-pub struct Image16BPP<'a, C: PixelColor> {
-    /// Image width
-    width: u32,
+pub type Image16BPP<'a, C> = Image<'a, C, ImageType16BPP>;
 
-    /// Image height
-    height: u32,
+/// 16 bits per pixel image type
+#[derive(Debug, Copy, Clone)]
+pub enum ImageType16BPP {}
```

Comment:
Can this be a `struct` instead? What's the reasoning behind using an `enum`?

Reply:
The main reason is  that I modeled it after the `Font...Conf` types, which are also using enums. If I understand it correctly the main advantage of an empty enum is that it can never be instantiated and is only used at type level.

Reply:
Woops, I missed that. Ok yeah keep these as enums then :slightly_smiling_face: makes sense for consistency.